### ISSUE TITLE
diff of array values generates operations in order which throws exception on diff-apply

### ISSUE THREAD

**Issue by zirkel** [angus-c/just#443]
### How to reproduce
```
const { diff } = require('just-diff')
const { diffApply } = require('just-diff-apply')

const obj1 = { numbers: [1, 2, 3, 4] };
const obj2 = { numbers: [5, 6] };
const changes = diff(obj1, obj2); // [{op: "remove", path: ["numbers", 2]}, {op: "remove", path: ["numbers", 3]}, …]
diffApply(obj1, changes); // throws Exception "expected to find property numbers,3 in object [object Object]"
```

**Comment by angus-c** [comment]
Good catch, thanks. I will fix

**Comment by angus-c** [comment]
Fixed in just-diff@5.0.2
